packages/preview/sdu-touying-simpl/1.0.0/example/main.typ

```diff
@@ -0,0 +1,830 @@
+#import "@preview/sdu-touying-simpl:1.0.0" :*
+// #import "../lib.typ" :*
+// https://typst.app/universe/package/timeliney
+#import "@preview/timeliney:0.2.1"
+
+// https://typst.app/universe/package/codly
+#import "@preview/codly:1.3.0": *
+#import "@preview/codly-languages:0.1.1": *
+#show: codly-init.with()
+
+#codly(languages: codly-languages)
+//https://typst.app/universe/package/fletcher
+#import "@preview/fletcher:0.5.7" as fletcher: diagram, node, edge
+
+//https://typst.app/universe/package/ctheorems
+// 数学定理相关
+#import "@preview/ctheorems:1.1.3": *
+#show: thmrules.with(qed-symbol: $square$) // 设置定理结束符号
+#let cdefinition = thmbox("definition", "Definition", inset: (x: 1.2em, top: 1em))
+
+#let cexample = thmplain("example", "Example").with(numbering: none)
+#let cproof = thmproof("proof", "Proof")
+
+#let ctheorem = thmbox("theorem", "Theorem", fill: rgb("#eeffee"))
+#let corollary = thmplain(
+  "corollary",
+  "Corollary",
+  base: "theorem",
+  titlefmt: strong
+)
+//https://typst.app/universe/package/showybox
+#import "@preview/showybox:2.0.4": showybox
+
+// https://typst.app/universe/package/gentle-clues
+#import "@preview/gentle-clues:1.2.0": *
+
+#set text(font: (
+  (
+    name: "Inria Serif",
+    covers: "latin-in-cjk",
+  ),
+  "Noto Serif CJK SC"
+))
+// #set text(font: "Source Han Serif", lang: "CN",region: "CN")
+#show: sdu-theme.with(
+  aspect-ratio: "16-9",
+   config-info(
+    title: [Typst template for Shandong University],
+    subtitle: [Continuously Improving...],
+    author: [孙更欣],
+    date: datetime.today(),
+    institution: [计算机科学与技术学院],
+    logo:sdu-logo
+  ),
+  footer-line-color: sdu-red,
+)
+#set heading(numbering: "1.1")
+#show heading.where(level: 1): set heading(numbering: "1.")
+#title-slide()
+#outline-slide([Outline])
+#let typst-color = rgb("#239DAD")
+
+#let Typst = text(fill: typst-color, weight: "bold", "Typst")
+
+#let Touying = text(fill: rgb("#425066"), weight: "bold", "Touying")
+
+#let Markdown = text(fill: rgb(purple), weight: "bold", "Markdown")
+
+
+#let TeX = {
+  set text(font: "New Computer Modern", weight: "regular")
+  box(width: 1.7em, {
+    [T]
+    place(top, dx: 0.56em, dy: 0.22em)[E]
+    place(top, dx: 1.1em)[X]
+  })
+}
+#import "@preview/octique:0.1.0": *
```

Comment:
This package uses the deprecated `image.decode` function which will be removed in a future release, and break this package. There is an [open PR](https://github.com/0x6b/typst-octique/pull/2) on the repo, but it doesn't seem to be actively maintained.
Since it is quite small, you could maybe vendor a fork of the `octique` package in this repository until either the PR is merged, or someone else picks it up.